app/lib/features/home/pages/dashboard.dart

```diff
@@ -116,15 +116,15 @@ class Dashboard extends ConsumerWidget {
               context: context,
               title: L10n.of(context).pins,
               iconData: Atlas.pin,
-              color: Colors.orangeAccent,
+              color: const Color(0xff7c4a4a),
```

Comment:
Why these colors are not from ```ThemeContext``` but hardcoded??

Reply:
I was testing a bit on colors so left as static. I will put it on Color Scheme class. Good notice!